package.json

```diff
@@ -2,7 +2,6 @@
   "name": "time-tracker",
   "version": "1.72.5",
   "scripts": {
-    "config": "ts-node ./scripts/setenv.ts",
```

Comment:
setenv.ts is only used here, I think we need to also delete this file from the project